tests/buffer_test.py

```diff
@@ -81,6 +75,106 @@ def test_config_buffer_profile_add_headroom_to_dynamic_profile(self):
         assert result.exit_code != 0
         assert "Can't change profile alpha_profile from dynamically calculating headroom to non-dynamically one" in result.output
 
+    def test_config_shp_size(self):
+        runner = CliRunner()
+        db = Db()
+        result = runner.invoke(config.config.commands["buffer"].commands["shared-headroom-pool"].commands["size"],
+                               ["200000"], obj=db)
+        print(result.exit_code)
+        print(result.output)
+        assert db.cfgdb.get_entry('BUFFER_POOL', 'ingress_lossless_pool') == {'mode': 'dynamic', 'type': 'ingress', 'xoff': '200000'}
+
+    def test_config_shp_size_negative(self):
+        runner = CliRunner()
+        result = runner.invoke(config.config.commands["buffer"].commands["shared-headroom-pool"].commands["size"],
+                               ["20000000"])
+        print(result.exit_code)
+        print(result.output)
+        assert result.exit_code != 0
+        assert "shared headroom pool must be less than mmu size" in result.output
+
+    def test_config_shp_ratio(self):
```

Comment:
add a negative test for ratio

Reply:
This one should not be removed. The `test_config_shp_size` can be removed instead.